lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -412,6 +413,24 @@ def add_check_manifest_lock_script_phase
           end
         end
 
+        # Adds a symlink at $PROJECT_DIR/Pods pointing to $PODS_ROOT if the user project is behind a symlink
+        #
+        # @note Replaces any existing symlinks at $PROJECT_DIR/Pods
+        #
+        # @return [void]
+        #
+        def add_sandbox_symlink
```

Comment:
since this has nothing to do with the particular target, it probably belongs in the `UserProjectIntegrator`

Reply:
If the Pod is a local development pod, I will create the symlink, too. So it is in `TargetIntegrator`.

Reply:
but this still has nothing to do with an individual _target_, only an individual _project_